README.md

```diff
@@ -22,7 +22,7 @@ For more information: http://logbook.readthedocs.org
 [ti]: https://secure.travis-ci.org/getlogbook/logbook.svg?branch=master
 [tl]: https://travis-ci.org/getlogbook/logbook
 [ai]: https://ci.appveyor.com/api/projects/status/quu99exa26e06npp?svg=true
-[vi]: https://img.shields.io/badge/python-2.6%2C2.7%2C3.3%2C3.4%2C3.5-green.svg
+[vi]: https://img.shields.io/badge/python-2.6%2C2.7%2C3.3%2C3.4%2C3.5%2C3.6%2C3.7-green.svg
```

Comment:
2.6 should be removed (not part of your change, but still while you're at it :-)